graphics/fpga-graphics/verilator/main_beam.cpp

```diff
@@ -39,19 +39,24 @@ int main(int argc, char* argv[]) {
         return 1;
     }
 
-    sdl_renderer = SDL_CreateRenderer(sdl_window, -1, SDL_RENDERER_ACCELERATED);
+    sdl_renderer = SDL_CreateRenderer(sdl_window, -1, SDL_RENDERER_ACCELERATED | SDL_RENDERER_PRESENTVSYNC);
     if (!sdl_renderer) {
         printf("Renderer creation failed: %s\n", SDL_GetError());
         return 1;
     }
 
-    sdl_texture = SDL_CreateTexture(sdl_renderer, SDL_PIXELFORMAT_RGBA8888,
+    sdl_texture = SDL_CreateTexture(sdl_renderer, SDL_PIXELFORMAT_ARGB8888,
```

Comment:
This was also a small perf gain. When I profiled the code on my Macs, SDL had to convert the pixels from `RGBA8888 `ARGB8888`, and this avoids that conversion. Though I was still getting 60 FPS without it, so not a big deal if you want to keep `RGBA8888` for Linux/Windows.

Reply:
The render looks fine in my Linux tests, but does give the warning: 
Window pixel format differs from render pixel format: SDL_PIXELFORMAT_RGB888

Reply:
Okay, I reverted back to `SDL_PIXELFORMAT_RGBA8888`. There's probably some way of matching the window pixel format without conversion, but I suspect that would require `SDL_TEXTUREACCESS_STREAMING`.